src/modules/loader.js

```diff
@@ -0,0 +1,16 @@
+const ulList = document.querySelector('.createdList');
+
+const list = [
+  { description: ['wash cups', 'goods for sale', 'clean the house', 'yello yellow'] },
+  { completed: [false, false, false, false] },
+  { index: [1, 2, 3, 4] },
+
+];
```

Comment:
- [ ] Nice work populating the tasks using array of object :+1: However It's required that each task object should contain three keys: `description [string]`, `completed [bool]`, and `index: [number]`. This array could look like this:
```sh
const list = [
  {
    description: 'A description about task 1',
    completed: false,
    index: 1,
  },
  {
    description: 'A description about task 2',
    completed: false,
    index: 2,
  },
  ...
];
```